File: HW2/Dellal_Zeynep_Doga.dart
void main() {
  print("--UNCONDITIONAL CONTINUE--");
  for (var i = 0; i < 5; i++) {
    print(i);
    continue;
  }

  print("--UNCONDITIONAL EXIT--");
  for (var i = 1; i < 5; i++) {
    print(i);
    break;
  }

  print("--UNLABELED CONTINUE--");
  for (var i = 0; i < 5; i++) {
    if (i == 3) {
      continue;
    }
    print(i);
  }

  print("--UNLABELED BREAK--");
  var i = 0;
  while (i < 5) {
    i++;
    print(i);
    if (i == 3) break;
  }

  print("--LABELED CONTINUE--");
  labelforcontinue:
  for (var i = 0; i <= 5; i++) {
    for (var j = 0; j < 3; j++) {
      if (i == j) {
        continue labelforcontinue;
      }
    }
    print(i);
  }

  print("--LABELED BREAK--");
  var a = 1;
  exitouterloop:
  while (a < 5) {
    while (a > 0) {
      print(a);
      if (a == 3) break exitouterloop;
      a++;
    }
  }
}

File: HW2/Dellal_Zeynep_Doga.rs
fn main() {
    println!("UNCONDITIONAL BREAK");
    let mut i = 1;
    while i < 5 {
        println!("{}", i);
        i = i + 1;
        break;
    }
    
    println!("UNCONDITIONAL CONTINUE");
    i = 1;
    while i < 5 {
        println!("{}", i);
        i = i + 1;
        continue;
    }
    
    println!("UNLABELED CONDITIONAL BREAK");
    i = 1;
    while i < 5 {
        if i == 3 {break;}
        println!("{}", i);
        i = i + 1;
    }
    
    println!("UNLABELED CONDITIONAL CONTINUE");
    i = 1;
    while i < 5 {
        i = i + 1;
        if i == 3 {continue;}
        println!("{}", i);
    }
    
     println!("LABELED BREAK");
    'label:for i in 1..8{ 
        println!("i is {}",i);
        for j in 3..8{
          if i==j {
            break 'label;
          }
      }
   }
   
    println!("LABELED CONTINUE");
    'labelforcontinue:for i in 1..5{ 
        for j in 1..5{
          if i==3 {
            continue 'labelforcontinue;
          }
      }
      println!("i is {}",i);
   }
}
